1091. Shortest Path in Binary Matrix/solution.py

```diff
@@ -4,23 +4,78 @@ class Solution:
     # cells containing a 0. Each cell is connected to the 8 cells around it
     # that either share an edge or a corner. If no path is found, return -1.
     def shortestPathBinaryMatrix(self, grid: List[List[int]]) -> int:
+        # Check that top left and bottom right cells contain 0s. Otherwise no clear
+        # path can exist, so return -1
+        gridHeight = len(grid)
+        gridWidth = len(grid[0])
+        
+        if grid[0][0] != 0 or grid[-1][-1] != 0:
+            return -1
+        
         # Use BFS to find the shortest path so keep track of nodes
         # to visit in a queue. Each entry is a tuple of the col and row index of
```

Comment:
I've commonly had confusions with saying the "col and row" index. Can you explain strictly what that means? I usually say "the index inside the column" or "the index inside the row".